atlas/schema.hcl

```diff
@@ -56,8 +142,8 @@ table "expense_snapshot_v1" {
 }
 table "expense_v1" {
   schema = schema.public
-  comment = "This table links expenses back to its project. No data should be stored here."
-  column "project_id" {
+  comment = "This table links expenses back to its receipt. No data should be stored here."
+  column "receipt_id" {
```

Comment:
Does the hierarchy look like project > receipt > expense then? What about project > expense > receipt instead? How does this set the precedence for how revenue is structured?

Reply:
I was thinking project > receipt > expense because in my head a receipt is a list of expenses. Also correct me if I'm wrong but receipts should have nothing to do with income - those would be invoices? The hierarchy for that would be project > invoice > income. From there revenue would be calculated based on joins or other BE calculations between the income and expenses. Wdyt?

Reply:
That makes total sense. Thanks for explaining it 🙏